events/foldercompletion.rst

```diff
@@ -6,13 +6,20 @@ contents for a folder changes. It contains the completion percentage for
 a given remote device and is emitted once per currently connected remote
 device.
 
+.. versionadded:: 1.20.0
+
+  Indication whether the remote device has accepted the folder (shares it with
+  us) as well, and whether it is paused.  The ``remoteState`` field is
+  meaningless when the remote device is not connected, otherwise it can be
+  either ``valid``, ``paused``, ``notSharing``, or ``unknown``.
```

Comment:
When it's not connected it's `unknown`, which while not very helpful is meaningful?  
I'd however make expolicit that `valid` means that the remote has accepted/is sharing the folder with us - from the term `valid` I'd not know what it means (it gets kind-of clear when excluding all other possibilities, but still).

Reply:
Fixed, hopefully.